plugins/kubernetes/src/api/types.ts

```diff
@@ -27,8 +28,9 @@ export const kubernetesApiRef = createApiRef<KubernetesApi>({
 });
 
 export interface KubernetesApi {
-  getObjectsByServiceId(
+  getObjectsByLabelSelector(
```

Comment:
Let's make this `getObjectsByEntity` instead and pass in the entire entity, instead of breaking it up client-side. That'll provide a lot more flexibility for doing further extensions like this in the future.

Reply:
@Rugvip got back around to this.